modules/ensembl/annotation/dumpanno/meta.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: "annotation_dumpanno"
+
+description: This module is used to dump the functional annotation of a genome.
+
+keywords:
+  - core_database
+  - ensembl-legacy-scripts
+  - fasta
+  - docker
+  - container
+  
+tools:
+  - "annotation":
+      homepage: "https://github.com/Ensembl/nextflow_modules"
+      description: "Module which function to dump annotation directly from a core database."
```

Comment:
Does anyone know for sure if the meta:description for each main.nf relates only to that specific main.nf OR should it relate to the parent folder 'annotation' in this case. If so I will need to update some of my ported modules as I used descriptions related to the parent folder

Reply:
As we have a description of the dump_Anno main functionality on line 4, while this description is under "annotation"

Reply:
I have checked the meta.yml for other modules in nf-core, I believe here it should describe what the tool annotation should do.